lambda-s3-translate-cdk/README.md

```diff
@@ -0,0 +1,36 @@
+# S3 Triggered Lambda Function that starts an Amazon Translate Job and stores result into another S3 bucket
+This repo contains serverless patterns showing how to setup a Lambda with an S3 *object created* trigger that starts a basic, Amazon Translate Job. The result is placed into another S3 bucket.
+
+![Demo Project Solution Architecture Diagram](architecture.png)
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* AWS Account
+* AWS CLI already configured with Administrator permission
+* [NodeJS 14.x installed](https://nodejs.org/en/download/)
+* CDK v2 installed: See Getting Started With the AWS CDK
+* Python CDK required libraries: (install with pip install -r requirements.txt)
+* Clone this repo!
+
+## Deployment Instructions
+
+1. Within your CDK Python module directory(where all your cdk stacks are located) create a constructs folder and within the constructs folder, create an assets folder.
```

Comment:
Not sure the deployment instructions are clear. Write the instructions based on deploying this CDK construct here or as a clean checkout of this pattern. 

Have a look at other examples if you get stuck